setup.cfg

```diff
@@ -31,11 +31,17 @@ universal = 1
 [tool:pytest]
 minversion = 3.1
 testpaths = tests
-mock_use_standalone_module = true
 filterwarnings =
     ignore::sqlalchemy.exc.SAWarning
 addopts =
     --appini=tests/test.ini
-    --cov=clld
-    --cov-report term-missing
+    --cov
+    --cov-report=term
+    --cov-report=html
+mock_use_standalone_module = true
+
+[coverage:run]
+source = clld
 
+[coverage:report]
+show_missing = true
```

Comment:
Ah ok. So we'll get missing lines after all, right?

Reply:
Yes, we can configure all of coverage in `[coverage:*]` sections of `setup.cfg` like this without needing to look up how to pass it via `pytest-cov`, so I just moved it.